docs/automations.md

```diff
@@ -31,7 +31,9 @@ from homeassistant.helpers.trigger import Trigger, TriggerActionRunner, TriggerC
 from homeassistant.helpers.typing import ConfigType
 
 _OPTIONS_SCHEMA = vol.Schema({
-    vol.Required("event_type"): cv.string,
+    vol.Required("behavior", default="any"): vol.In(
+        ["first", "last", "any"]
+    ),
 })
```

Comment:
I feel like we should adapt this trigger to be more realistic, I'd love for the example YAML to adhere to the example trigger, but from the example it wasn't clear what `event_type: str` was used for